templates/monster_weapons.html

```diff
@@ -32,29 +43,46 @@ <h3 id="breacherheading" class="card-title">{{weapon['name']}}</h3>
         					<div class="card-body">
         						<p> {{weapon['description']}} </p>
                                 <div class="row">
-        						<dl class="col-lg-4 col-md-6 col-sm-12">
+        						<dl class="col-lg-5 col-md-6 col-sm-12">
         							<dt>Damage</dt><dd>{{weapon['damage']}}</dd>
-                                    <dt>Armor Pierce Level</dt><dd>{{weapon['ap_level']}}</dd>
-                                    <dt>Reflex Modifier</dt><dd>{{weapon['reflex_modifier']}}</dd>
-                                    <dt>Move Speed Penalty</dt><dd>{{weapon['move_speed_penalty']}}</dd>
-                                    <dt>Suppression Level</dt><dd>{{weapon['suppression_level']}}</dd>
+                                    <dt>APL</dt><dd>{{weapon['ap_level']}}</dd>
+                                    <dt>RMod</dt><dd>{{weapon['reflex_modifier']}}</dd>
+                                    <dt>MSP</dt><dd>{{weapon['move_speed_penalty']}}</dd>
+                                    <dt>Sup Lvl</dt><dd>{{weapon['suppression_level']}}</dd>
         							<dt>Type</dt><dd>{{weapon['type']}}</dd>
         						</dl>
-                                <dl class="col-lg-4 col-md-6 col-sm-12">
+                                <dl class="col-lg-5 col-md-6 col-sm-12">
                                     <dt>Weapon Cost</dt><dd> ${{weapon['cost']}}</dd>
                                     <dt>Mag Cost</dt><dd>${{weapon['mag_cost']}}</dd>
-                                    <dt>Mag Capacity</dt><dd>{{weapon['capacity']}}rds</dd>
-                                    <dt>Reload DC</dt><dd>{{weapon['reload_dc']}}</dd>
-                                </dl>
-                                <dl class="col-lg-4 col-md-6 col-sm-12">
-                                    <dt>Range 1</dt><dd>{{weapon['r1']}}</dd>
-                                    <dt>Range 2</dt><dd>{{weapon['r2']}}</dd>
-                                    <dt>Range 3</dt><dd>{{weapon['r3']}}</dd>
-                                    <dt>Accuracy 1</dt><dd>{{weapon['acc1']}}</dd>
-                                    <dt>Accuracy 2</dt><dd>{{weapon['acc2']}}</dd>
-                                    <dt>Accuracy 3</dt><dd>{{weapon['acc3']}}</dd>
+                                    <dt>Cap</dt><dd>{{weapon['capacity']}}rds</dd>
+                                    <dt>RDC</dt><dd>{{weapon['reload_dc']}}</dd>
+                                    <dt>AFR:</dt> <dd> {{weapon['auto_fire_rate']}} rnds/Act </dd>
```

Comment:
Anomalous whitespace in this line

Reply:
Wanted a space between the number and the unit. I think it looked nicer. Probably should do the same thing with line 57...